spec/Cabin/SingleConfigurableLight.vspec

```diff
@@ -0,0 +1,29 @@
+#
+# (C) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+#
+# Generic specification for a light whose color and brightness can be configured.
+#
+
+IsLightOn:
+  type: actuator
+  datatype: boolean
+  description: Indicates whether the light is turned on. True = On, False = Off.
+
+Intensity:
+  type: actuator
+  datatype: uint8
+  unit: percent
+  min: 1
+  max: 100
+  description: How much of the maximum possible brightness of the light. is used. 1 = Maximum attenuation, 100 = no attenuation (i.e., full brightness).
+  comment: Attenuation = 100 - Intensity.
+
+Color:
+  type: actuator
+  datatype: string
+  description: Hexadecimal color code.
```

Comment:
Here I would prefer if we can refer to standard/spec/wiki on how the color code shall be formatted, or just describe it. Like `Color shall be given as a 3 byte RGB value preceded by a hash, allowed range #000000 to #FFFFFF` and potentially include an example like `Examples: #C0C0C0 = Silver, #FFD700 = Gold`

One could of course argue that a uint32 (with range) would work as well.

Reply:
Hexadecimal value is not part of the [supported datatypes in VSS](https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/data_types/). I think simply defining it as an integer with base 10 can cause confusion as the user would be responsible for making the conversion. 

Another option is to have `Color.Red`, `Color.Green`, `Color.Blue` with min and max values (i.e., from 0 to 255). However, I think that approach would be too verbose. 

As for now, we can keep the hexadecimal representation with string datatype. I will add the recommended comments in the definition.